src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+function addBodyMessage(message) {
+  document.body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification">
+      ${message}
+    </div>
+  `);
+}
+
+function checkClick(resolve) {
+  document.addEventListener('click', resolve);
+}
+
+function checkContextmenu(resolve) {
```

Comment:
small nit: `checkContextmenu` -> `checkContextMenu`
(do not forget to write in camalCase)